build/reply-handle-new/block.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "reply-handler",
```

Comment:
should we use `reply-intent` as (folder) name because this seems to be the commonly used name?

* https://developer.x.com/en/docs/twitter-for-websites/tweet-button/guides/web-intent
* https://developer.android.com/training/sharing/send